src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStep.java

```diff
@@ -261,35 +287,11 @@ public boolean getShowGitToolOptions() {
             return false;
         }
 
-        @Inject
```

Comment:
This is all unused, actually leftover from the SCM refactor, just cleaning this up now